test-tools/src/main/java/org/terracotta/utilities/test/AbstractRepeatingTest.java

```diff
@@ -52,10 +55,71 @@
  * }
  * </code></pre>
  *
- * The {@code Request} can be composed of any collection of tests supported by the methods in the {@code Request} class.
+ * While usually used to repeat a single test, the {@code Request} can be composed of any collection of tests
+ * supported by the methods in the {@code Request} class.
+ * <p>
+ * <b>Each variant of the {@code repeatTestRequest} method <i>substitutes</i> "capture" files for {@code System.out}
+ * and {@code System.err} prior to running the test(s).  Tests relying on output to {@code System.out} or
+ * {@code System.err} may be incompatible with this method.</b>
+ * <p>
+ * With each variant of the {@code repeatTestRequest} method, when a test fails:
+ * <ol>
+ *   <li>The JUnit failure summary is emitted to {@code System.out}.</li>
+ *   <li>The captured {@code stdout} content for the failing test(s) is copied to {@code System.out};
+ *      this output is wrapped in marker lines <pre>{@code
+ *        >>> Begin failing stdout
+ *        ...
+ *        <<< End failing stdout
+ *      }</pre></li>
+ *   <li>The captured {@code stderr} content for the failing test(s) is copies to {@code System.err};
+ *       this output is wrapped in marker lines <pre>{@code
+ *         >>> Begin failing stderr
+ *         ...
+ *         <<< End failing stderr
+ *       }</pre></li>
+ *   <li>An {@code AssertionError} is thrown with the failure exception for each failing test present
+ *      as a suppressed exception.</li>
+ * </ol>
+ * <p>
+ * For {@code repeatTestRequest} variants <b>not</b> accepting an {@code AbstractRepeatingRunListener} instance,
+ * each iteration of the test emits progress output to {@code System.out} for each JUnit {@code RunListener} event
+ * raised.  To control that output, use a {@code repeatTestRequest} variant accepting a
+ * <code>Set&lt;{@link ListenerEvents}&gt;</code> or {@link AbstractRepeatingRunListener} instance.
+ * <p>
+ * Variants of {@code repeatTestRequest} accepting a {@link Monitor} instance support pre- and post- test
+ * activities.  While a {@code Monitor} instance does not have access to the JUnit {@code Runner} instances
+ * created for the tests being run, a {@code Monitor} instance can use used for diagnostics.  As an example,
```

Comment:
Minor changes required in Java doc, replace `instance can use used` to instance can be used`

Reply:
Updated.  I also discovered a javadoc glitch that I corrected.